docs/docs.md

```diff
@@ -13,11 +13,14 @@ from dynamics import DynamicsClient
 - client_id: str - Client id (e.g. UUID).
 - client_secret: str - Client secret (e.g. OAuth password).
 - scope: List[str] - List of urls in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.
+- *resource: Optional[str] - Url in form: 'https://[Organization URI]/scope'. Defines the database records that the API connection has access to.*
 
 Establish a Microsoft Dynamics 365 Dataverse API client connection
 using OAuth 2.0 Client Credentials Flow. Client Credentials require an application user to be
 created in Dynamics, and granting it an appropriate security role.
 
+*If you are experiencing auth errors, inspect the returned auth token's `aud` and see whether it resolves to `00000002-0000-0000-c000-000000000000` instead of your CRM url. If so, you may want to specify the resource URL using the optional 'resource' parameter.*
```

Comment:
This instruction is great, but could you break it up to multiple lines at about the 100 character mark? Makes it easier to read on bigger monitors without adjusting the window size.

Reply:
Done